docs/content/a-tale-of-3-nightclubs.md

```diff
@@ -108,4 +108,86 @@ We can even use pattern matching on the result:
         });
 
 The thing to note here is how the Validations can be composed together in a computation expression.
-The type system is making sure that failures flow through your computation in a safe manner.
\ No newline at end of file
+The type system is making sure that failures flow through your computation in a safe manner.
+
+## Part Two : Club Tropicana
+
+Part One showed monadic composition, which from the perspective of Validation is *fail-fast*. That is, any failed check short-circuits subsequent checks. This nicely models nightclubs in the real world, as anyone who has dashed home for a pair of smart shoes and returned, only to be told that your tie does not pass muster, will attest.
+
+But what about an ideal nightclub? One that tells you *everything* that is wrong with you.
+
+Applicative functors to the rescue!
+
+Let's compose some validation checks that accumulate failures using LINQ sugar:
+
+    [lang=csharp]
+    class ClubTropicana
+    {
+        public static Result<decimal, string> CostToEnter(Person p)
+        {
+            return from c in Club.CheckAge(p)
+                    join x in Club.CheckClothes(p) on 1 equals 1
+                    join y in Club.CheckSobriety(p) on 1 equals 1
+                    select c.Gender == Gender.Female ? 0m : 7.5m;
+        }
+    }
+
+And the use? Dave tried the second nightclub after a few more drinks in the pub:
```

Comment:
I guess it could be more explicit. Would changing it to:

> The usage is the same as above except that as a result we will get either a success or a list of accumulated error messages from all the checks.

make more sense to you?